src/python/pants/vcs/changed.py

```diff
@@ -104,11 +104,17 @@ class ChangedOptions:
 
     since: str | None
     diffspec: str | None
-    dependees: DependeesOption
+    dependents: DependentsOption
 
     @classmethod
     def from_options(cls, options: OptionValueContainer) -> ChangedOptions:
-        return cls(options.since, options.diffspec, options.dependees)
+        return cls(
```

Comment:
This (and the option deprecation) is the only part of the diff that isn't a straight find-and-replace